packages/chrome/src/elements/header/HeaderItem.spec.tsx

```diff
@@ -54,8 +54,8 @@ describe('HeaderItem', () => {
       talk: PALETTE.product.talk
     };
 
-    PRODUCTS.forEach(product => {
```

Comment:
Tests refactoring removes expressions from the `describe` block scope.

https://github.com/jest-community/eslint-plugin-jest/blob/v24.7.0/docs/rules/require-hook.md#rule-details